### PR TITLE
Using new table format and show mem usage for hosts and services

### PR DESCRIPTION
**Pull request by smousa**

### CODE REVIEW

File: cli/api/service.go [hunk 1]
```diff
@@ -77,33 +82,121 @@ func (a *api) GetServices() ([]service.Service, error) {
...
+				row["Status"] = stat.Status.String()
+				row["Hostname"] = hostmap[stat.State.HostID]
+				row["DockerID"] = fmt.Sprintf("%.12s", stat.State.DockerID)
+				row["Uptime"] = stat.State.Uptime().String()
+
+				if stat.State.InSync {
+					row["InSync"] = "Y"
+				} else {
+					row["InSync"] = "N"
+				}
```
Comment (wwwtyro) [inline cli/api/service.go]: Could shorten this with:

```
row["InSync"] = "N"
if stat.State.InSync {
    row["InSync"] = "Y"
}
```

File: cli/api/service.go [hunk 2]
```diff
@@ -77,33 +82,121 @@ func (a *api) GetServices() ([]service.Service, error) {
...
+				if svc.Instances > 1 {
+					row["Name"] = fmt.Sprintf("%s/%d", svc.Name, stat.State.InstanceID)
+				} else {
+					row["Name"] = svc.Name
+				}
```
Comment (wwwtyro) [inline cli/api/service.go]: same here with Name

**Comment by wwwtyro** [comment]
Summer promised to fill out the paperwork for bytefmt.

**Comment by tgjonestx** [comment]
See Jira [CC-1000](https://jira.zenoss.com/browse/CC-1000)

**Pull request merged**